sdk/monitor/opentelemetry-exporter-azure-monitor/src/platform/nodejs/persist/fileSystemPersist.ts

```diff
@@ -4,8 +4,7 @@
 import * as fs from "fs";
 import * as os from "os";
 import * as path from "path";
-import { Logger } from "@opentelemetry/api";
-import { NoopLogger } from "@opentelemetry/core";
+import { Logger, NoopLogger } from "@opentelemetry/api";
```

Comment:
Note: My diag-logger will affect these in a future release where they will disappear